module/importer-magento-1/src/Infrastructure/Model/ProductModel.php

```diff
@@ -12,16 +12,60 @@
  */
 class ProductModel
 {
+    /**
+     * @var string
+     */
+    private string $sku;
+
+    /**
+     * @var string
+     */
+    private string $type;
+
+    /**
+     * @var string
+     */
+    private string $template;
+
     /**
      * @var array
      */
     private array $versions;
 
     /**
+     * @param string $sku
+     * @param string $type
+     * @param string $template
+     */
+    public function __construct(string $sku, string $type, string $template)
+    {
+        $this->sku = $sku;
```

Comment:
Versions are no longer initialized as an array. Can cause error in i.e. :: has()